webpack/assets/javascripts/foreman_tools.js

```diff
@@ -133,6 +133,4 @@ export function updateTable(element) {
 }
 
 // generates an absolute, needed in case of running Foreman from a subpath
-export function foremanUrl(path) {
-  return window.URL_PREFIX + path;
-}
+export { foremanUrl } from './react_app/common/helpers';
```

Comment:
it's declared in `/react_app/common/helpers` so plugins that point to that folder could run their tests.
`jest` can't access to files outside the scope of the folder so it fails.